roles/ceph-mon/tasks/docker/checks.yml

```diff
@@ -13,7 +13,7 @@
 - name: stat for ceph config and keys
   stat:
     path: "{{ item }}"
-  with_items: ceph_config_keys
+  with_items: "{{ ceph_config_keys }}"
```

Comment:
You can also use `default([])` inline. This has the effect of [skipping the task](http://docs.ansible.com/ansible/playbooks_conditionals.html#loops-and-conditionals) if the variable is undefined.

```yaml
with_items: " {{ cepth_config_keys | default([]) }}"
```